### PR TITLE
Spark Release 2.9.0-2.4.3 site docs and release notes

### PR DESCRIPTION
**Pull request by akirillov**
## Jira Ticket
[DCOS-54712: Release docs for Spark 2.9.0-2.4.3](https://jira.mesosphere.com/browse/DCOS-54712)

## Description of changes being made
Added documentation for the new version of DCOS Spark. Mostly copy-paste from a previous version with the exception of Release Notes and version used across the documentation.

## Checklist
- [x] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [x] Test all commands and procedures where applicable.
- [x] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/mesosphere/dcos/services/spark/2.9.0-2.4.3/release-notes/index.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+render: mustache
+model: /mesosphere/dcos/services/spark/data.yml
+---
+
+## Version Spark and Spark history 2.9.0-2.4.3
+
+### New features
+- Marathon group role enforcement support for DCOS 1.14
+- Role propagation from Dispatcher to all submitted Spark applications
+- Dispatcher role enforcement for submitted Spark applications
```
Comment (samvantran) [inline pages/mesosphere/dcos/services/spark/2.9.0-2.4.3/release-notes/index.md]: Did we add any new docs to explain these features in our service guide? Would be good to have some information/examples so users know what they need to submit to enable new features.

Comment (akirillov) [reply pages/mesosphere/dcos/services/spark/2.9.0-2.4.3/release-notes/index.md]: Let's add node draining to fault-tolerance section (under Driver) and quota to installation section? WDYT?

**Comment by akirillov** [comment]
Thanks, @samvantran. Updated PR based on your comments. Added draining under `fault-tolerance` in `Driver` section. Added quota to `Custom installation` section in `install.md`.

**Comment by akirillov** [comment]
Thanks for the suggestions @samvantran. Committed suggested changes.

**Pull request merged**